docs/day-2-operations/Backups.md

```diff
@@ -0,0 +1,272 @@
+* `backend-redis`: Redis database (`backend-redis-storage`)
+* `system-redis`: Redis database (`system-redis-storage`)
+
+### `system-mysql`
+This is a relational database for storing the information about users, accounts, APIs, plans etc in the 3scale Admin Console. 
+
+A subset of this information related to Services is synchronized to the `Backend` component and stored in `backend-redis`. `system-mysql` is the source of truth for this information. 
+
+### `system-storage`
+This is for storing files to be read and written by the `System` component. They fall into two categories:
+
+* Configuration files read by the System component at run-time
+* Static files (HTML, CSS, JS, etc) uploaded to System by it's CMS feature, for the purpose of creating a Developer Portal.
+
+Note that `System` can be scaled horizontally with multiple pods uploading and reading said static files, hence the need for a `RWX` PersistentVolume.
+
+### `zync-database`
+This is a relational database for storing information related to the synchronization of identities between 3scale and an Identity provider.
+This information is not duplicated in other components and this is the sole source of truth.
+
+### `backend-redis`
+This contains multiple data sets used by the `Backend` component:
+
+* Usages: This is API usage information aggregated by `Backend`. It is used by `Backend` for rate-limiting decisions and by `System` to display analytics information in the UI or via API.
+* Config: This is configuration information about Services, Rate-limits, etc that is synchronized from `System` via an internal API. This is NOT the source of truth of this info, `System` and `system-mysql` is.
+* AuthKeys: Storage of OAuth keys created directly in `Backend`. This is the source of truth for this information.
+* Queues: Queues of Background Jobs to be executed by worker processses. These are ephemeral and are deleted once processed.
+
+### `system-redis`
+This contains Queues for jobs to be processed in background. These are ephemeral and are deleted once processed.
+
+## Backup Procedures
+
+### `system-mysql`
+
+1. Execute MySQL Backup Command
+   ```
+   oc rsh $(oc get pods -l 'deploymentConfig=system-mysql' -o json | jq -r '.items[0].metadata.name') bash -c 'export MYSQL_PWD=${MYSQL_ROOT_PASSWORD}; mysqldump --single-transaction -hsystem-mysql -uroot system' | gzip > system-mysql-backup.gz
+   ```
+
+### `system-storage`
+
+1. Archive the system-storage files to another storage.
+    ```
+    oc rsync $(oc get pods -l 'deploymentConfig=system-app' -o json | jq '.items[0].metadata.name' -r):/opt/system/public/system ./local/dir
+    ```
+
+### `zync-database`
+
+1. Execute Postgres Backup Command
+   ```
+    oc rsh $(oc get pods -l 'deploymentConfig=zync-database' -o json | jq '.items[0].metadata.name' -r) bash -c 'pg_dumpall -c --if-exists' | gzip > zync-database-backup.gz
```

Comment:
You may wish to date stamp these output files, if backups are to be run regularly